docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-setup.md

```diff
@@ -137,19 +131,43 @@ The **xCluster Disaster Recovery** tab also lists all the tables in replication
 
 - To find out the replication lag for a specific table, click the graph icon corresponding to that table.
 
-- To check if the replication has been properly configured for a table, check the status. If properly configured, the table's replication status is shown as _Operational_.
+- To delete a table from the replication, click **... > Remove Table**. This removes both the table and its index tables from replication. If you decide to remove an index table from the replication group, it does not remove its main table from the replication group.
+
+- Use the search bar to filter the view by table name, database, size, and more.
 
-    The status will be _Not Reported_ momentarily after the replication configuration is created until metrics are available for the replication configuration. This should take about 10 seconds.
+#### Status
 
-    If the replication lag increases beyond maximum acceptable lag defined during the replication setup or the lag is not being reported, the table's status is shown as _Warning_.
+To check if the replication has been properly configured for a table, check the status. If properly configured, the table's replication status is shown as _Operational_.
 
-    If the replication lag has increased so much that resuming or continuing replication cannot be accomplished via WAL logs but instead requires making another full copy from DR primary to DR replica, the status is shown as _Error: the table's replication stream is broken_, and [restarting the replication](#restart-replication) is required for those tables. If a lag alert is enabled on the replication, you are notified when the lag is behind the specified limit, in which case you may open the table on the replication view to check if any of these tables have their replication status as Error.
+The status will be _Not Reported_ momentarily after the replication configuration is created until metrics are available for the replication configuration. This should take about 10 seconds.
 
-    If YugabyteDB Anywhere is unable to obtain the status (for example, due to a heavy workload being run on the universe) the status for that table will be _UnableToFetch_. You may refresh the page to retry gathering information.
+If the replication lag has increased so much that resuming or continuing replication cannot be accomplished via WAL logs but instead requires making another full copy from DR primary to DR replica, the status is shown as _Missing op ID_, and you must [restart replication](#restart-replication) for those tables. If a lag alert is enabled on the replication, you are notified when the lag is behind the [replication lag alert](#set-up-replication-lag-alerts) threshold; if the replication stream is not yet broken and the lag is due to some other issues, the status is shown as _Warning_.
```

Comment:
It will follow the same pattern as the other alerts in YBA. The way that the user is notified depends on the channel that they have configured in http://<yba-address>/admin/alertConfig/notificationChannels
The alert will be visible on the Alerts page as well.